client/src/EstimatesComparison/EstimatesExplorerComponent.js

```diff
@@ -11,15 +12,23 @@ import {
   CheckBox,
   Details,
 } from "src/components/index";
+import { create_text_maker_component } from "src/components/misc_util_components";
+
+import common_subapp_faq from "src/components/PinnedFAQ/common_faq_questions.yaml";
+import est_faq from "src/components/PinnedFAQ/estimates_comparison_questions.yaml";
```

Comment:
Keeping the subapp-specific faq yaml files in PinnedFAQ because they shouldn't be needed in the infographics

Reply:
Should they move to the specific sub app's dir then?

Reply:
Or, since this yaml file's still empty, could just delete it. I'd prefer cleaning up those placeholder files before merging.